R/analyse_functional_network.R

```diff
@@ -147,36 +164,31 @@ does not match the number of rows in your data.", prefix = "\n", initial = ""))
   payload_id <- NULL
 
   if (!missing(binds_treatment)) {
-    if (missing(halo_color)) {
-      coloring <- input %>%
-        dplyr::filter({{ binds_treatment }}) %>%
-        dplyr::mutate(color = "#5680C1")
-    } else {
-      coloring <- input %>%
-        dplyr::filter({{ binds_treatment }}) %>%
-        dplyr::mutate(color = halo_color)
-    }
+    coloring <- input %>%
+      dplyr::filter({{ binds_treatment }}) %>%
+      dplyr::mutate(color = if (missing(halo_color)) "#5680C1" else halo_color)
```

Comment:
You cannot use the `missing()` function within `mutate()`. I think it is because `halo_color` is treated as a column name in this case and not as an argument. Not sure if there is a way around this.

Reply:
yes saw it - didnt had time to check the guthub action before  thanks!